policy/modules/services/redis.te

```diff
@@ -49,6 +49,8 @@ files_search_var_lib(redis_t)
 manage_dirs_pattern(redis_t, redis_runtime_t, redis_runtime_t)
 manage_files_pattern(redis_t, redis_runtime_t, redis_runtime_t)
 manage_lnk_files_pattern(redis_t, redis_runtime_t, redis_runtime_t)
+manage_sock_files_pattern(redis_t, redis_runtime_t, redis_runtime_t)
+files_runtime_filetrans(redis_t, redis_runtime_t, { dir sock_file })
```

Comment:
Any reason not have `dir file lnk_file` too?

Reply:
I see no harm in adding the access for those objects too. I didn't add them initially as I didn't see any AVCs related to them at the time.